compiler/luci-interpreter/src/kernels/Transpose.cpp

```diff
@@ -39,7 +39,7 @@ void Transpose::configure()
   int dims = input()->shape().num_dims();
   const int32_t *perm_data = getTensorData<int32_t>(perm());
 
-  assert(input()->shape().num_dims() <= 4);
+  assert(input()->shape().num_dims() <= 5);
```

Comment:
Already [supported in TF 2.8](https://github.com/tensorflow/tensorflow/blob/r2.8/tensorflow/lite/kernels/transpose.cc#L78) (implemented in https://github.com/tensorflow/tensorflow/commit/97b63ed8769d09a70a6617b067e63a12c7f8e071)